recipes-bsp/barebox/barebox.inc

```diff
@@ -82,6 +82,21 @@ do_install() {
 
 # Suffix that allows to create different barebox images in one BSP
 BAREBOX_IMAGE_SUFFIX = ""
+# This allows a machine.conf to specify the required images to install instead
+# of installing all found images
+BAREBOX_IMAGES ??= ""
```

Comment:
Couldn't this be `${B}/images/*.img` instead as default? That should simplify the condition further down.

Reply:
Good point. Changed it according your suggestion.